src/api-reference/budget/getting-started.md

```diff
@@ -66,7 +66,7 @@ Examples:
 HTTPie:
```

Comment:
There should be no links for specific tooling. Please follow the example in the [Documentation Template](https://github.concur.com/developer/api/blob/master/documentation-template.md).